dapps/shop/backend/queues/makeOfferProcessor.js

```diff
@@ -0,0 +1,127 @@
+const queues = require('./queues')
+
+const Web3 = require('web3')
```

Comment:
nit: I'd love for us to get rid of Web3 and replace it with ether.js. We don't have to do this in this PR but would be awesome to clean up later... :)